Deployment_Strategies/Rolling-Update-Deployment/rolling-update-svc.yml

```
@@ -0,0 +1,14 @@
+metadata:
+  name: rolling-update-service
+  namespace: rolling-ns
+spec:
+  selector:
+    app: online-shop
+  type: NodePort
+  ports:
+    - protocol: TCP
+      port: 3000
+      targetPort: 3000
+      nodePort: 30000
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)